package.json

```
@@ -41,6 +41,7 @@
     "yargs": "^17.3.1"
   },
   "devDependencies": {
+    "@fiatconnect/fiatconnect-types": "2.0.0",
```

```suggestion
    "@fiatconnect/fiatconnect-types": "^2.0.0",
```
If you take this, remember to update the lock file

---

good call!